src/main/kotlin/org/ionproject/integration/builder/implementations/IselTimetableTeachersBuilder.kt

```diff
@@ -93,11 +94,13 @@ class IselTimetableTeachersBuilder() : ITimetableTeachersBuilder<RawData> {
         timetable.programme = Programme(name = programme)
         timetable.calendarTerm = calendarTerm
         timetable.calendarSection = classSection
+        timetable.language = "pt-PT"
 
         courseTeacher.school = School(name = school)
         courseTeacher.programme = Programme(name = programme)
         courseTeacher.calendarTerm = calendarTerm
         courseTeacher.calendarSection = classSection
+        courseTeacher.language = "pt-PT"
```

Comment:
The values which identify languages will be the same for all write API operations and [they are listed here](https://github.com/i-on-project/core/blob/master/docs/api/write/insertClassSectionEvents.md#constants).